### ISSUE TITLE
Make new release on crates.io

### ISSUE THREAD

**Issue by LLFourn** [paritytech/libsecp256k1#50]
The version on github has several changes I've been using via git dependency. It would be nice to have them on crates.io.

**Comment by prestwich** [comment]
Seconded

**Comment by thomaseizinger** [comment]
https://github.com/libp2p/rust-libp2p/pull/1981 is also waiting on a new release.

**Comment by sorpaas** [comment]
We'll release 0.6.0 shortly. But I'll close this issue for now since it's really outdated.

**Comment by prestwich** [comment]
> We'll release 0.6.0 shortly. But I'll close this issue for now since it's really outdated.

so is the present release
